run.sh

```diff
@@ -32,7 +32,7 @@ if [ ! -d "$INPUT_REPOSITORY_PATH" ]
 then
 	echo "🐛 The given path is not a directory: ${INPUT_REPOSITORY_PATH}"
 	echo "🧰 To resolve this issue, set the 'repository-path' parameter to the directory that contains the local git clone of your repository."
-	exit 1
+	exit 0
```

Comment:
If the workflow instructs the action to look for the repository at a specific path using the [`repository-path`](https://github.com/buildpulse/buildpulse-action/blob/v0.10.0/README.md#repository-path) input, and there is no directory at that location, that's almost certainly an error that the user would want to know about. I think we should continue to exit with a nonzero exit code here.